tests/test_simplelog.py

```diff
@@ -0,0 +1,192 @@
+import sys
+
+import re
+import click
+import logging
+
+log_stream = None
+
+
+def tool_resetlog():
```

Comment:
Please see commit 86ea179 - that was the way I originally wanted to test, but it doesn't work at my site.